vendor/download.rb

```diff
@@ -31,6 +31,16 @@ def download(url, output_path=nil)
   end
 end
 
+def git_clone_archive(url, tag, archive_path)
+  clone_dir = archive_path.sub(/\.tar\..*\z/, "")
```

Comment:
```suggestion
  clone_dir = File.basename(archive_path, ".tar.gz")
```

`tar`で`-z`を使っているから`.tar.gz`決め打ちでいいよ。